portfolio/src/main/webapp/script.js

```diff
@@ -39,21 +39,20 @@ function randQuote() {
 // Fetch comments from DataServlet.java
 function getComments() {
 
-  fetch('/comments').then(response => response.json()).then((comments) => {
+  fetch('/comments').then(response => response.json()).then((cmtContainer) => {
 
-    var str_comments = "<h2>Comments</h2><p>";
+    if(!cmtContainer) return;
 
-    for (var index in comments){
-      str_comments += comments[index] + "<br>";
+    var str_comments = "<h2>Comments</h2>";
+
+    for (var index in cmtContainer) {
+      str_comments += ("<li>" + cmtContainer[index] + "</li>");
```

Comment:
Same as for the `<h2>` above: prefer creating individual `li` nodes and appending them to an `ol` or `ul`, then attaching that to the DOM once it's populated.